### DISCUSSION TITLE
Populate Top Tags (/tags) Page With Supported Tags

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16261] [Feature Requests]
Currently the logic for the Top Tags page (e.g. https://dev.to/tags) is hard to follow and the tags on the page are not able to be controlled by admin.

We should update /tags so that it only includes "supported tags", this way Forem admin are able to have some sort of control over the tags that appear there.

Note: we're really encouraging Forem Creators to set up supported tags, so this would only help add to it's importance.

**Comment by jeremyf** [comment]
Also I think this is a great feature.  And super quick to add.  But, I would want to consider a mechanism for showing creators "all tags that aren't supported but referenced".

Which certainly inter-relates with the search/filtering of https://github.com/forem/forem/discussions/16263

**Comment by ludwiczakpawel** [comment]
In relation to #16263 - this could be default sorting maybe...? So sorting options would be:
- **Supported** (default) - which is a silly name...
- **Relevance** (just myabe) - which would be current sorting.
- **Popularity** - sorted by number of posts
- **Name** - ¯\\\_(ツ)_/¯
- **Hotness** (just maybe.. a random idea) - sorted by number popularity (number of posts added) but in a given period of time (day? week?)

We should also think about a case when admin didn't flag any tags as "supported" OR they have 1000 tags but only 1 is supported..? That would make this page look weird.

**Comment by ildi-x** [comment]
I also find it strange that tags with 0 posts appear in the /tags page. This is more noticible within smaller Forem communities. 

For example on my Forem community the page https://www.1vibe.com/tags/ displays the _addicted_ tag but when going to https://www.1vibe.com/t/addicted you get a 404 error. There is only one post on 1VIBE that appears as a result if you search for the term _addicted_ https://www.1vibe.com/search?q=addicted&filters=class_name:Article and at the moment that post does not contain the _addicted_ tag. It's possible that I may have added this tag when I originally made the post and then removed it after. But this change should be reflected in the /tag page. 

Sometimes when writing a new post and enterting tags you may make a spelling mistake, for example instead of writing "music" you enter "muisc". If you recognize the mistake after publishing the post and go back and remove the tag from the post, it seems like Forem has already created the https://www.1vibe.com/t/muisc tag and at the moment there is no way for an admin to permanently delete tags. This tag which is a spelling mistake also continues to show up on the /tag page. 

Two other tags that have 0 posts and appear on /tags which show as 404 error when visiting the url: 

https://www.1vibe.com/t/cookinsoul
https://www.1vibe.com/t/soundfield

I think it's really important for admins to be able to delete tags or edit them instead of having to use the "Alias for" feature to reference to the correctly spelled tag when you make a spelling mistake. It's also bad logic for tags with 0 posts to appear in the /tags page, Im not sure if this is a bug and maybe an unintended action.

**Reply by ellativity** [reply]
+1 for being able to delete tags
